### PR TITLE
git test

### PR DESCRIPTION
**Pull request by ngraham20**
Changed readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,3 +2,5 @@
 A project to look at various ways to solve a maze
 
 Python or something will be used
+
+Git test
```
Comment (ngraham20) [inline README.md:6]: comment

Comment (ngraham20) [inline README.md:6]: review

**Review by ngraham20 — COMMENTED** [review]
Can you approve this?

**Review by kellyolivier — APPROVED** [review]
Kelly approves this message

**Pull request merged**